RecCalorimeter/src/components/TopoCaloNoisyCells.cpp

```diff
@@ -44,7 +44,7 @@ StatusCode TopoCaloNoisyCells::initialize() {
   double readNoisyCells;
   double readNoisyCellsOffset;
   tree->SetBranchAddress("cellId", &readCellId);
-  tree->SetBranchAddress("noiseLevel", &readNoisyCells);
+  tree->SetBranchAddress("noiseLevel", &readNoisyCells); // would be better to call branch noiseRMS rather than noiseLevel
```

Comment:
What is blocking to do it?

Reply:
I am not sure, this would mean that the noise map files have to be regenerated changing name to the rms branch. I would be ok with that but don’t want to disrupt the user who have already noise files at hand. What do you and @BrieucF prefer?

Reply:
Yes, we can handle this in another PR